content/en/docs/tasks/access-kubernetes-api/configure-aggregation-layer.md

```diff
@@ -246,6 +246,9 @@ spec:
   caBundle: <pem encoded ca cert that signs the server cert used by the webhook>
 ```
 
+The name of an APIService object must be a valid
+[path segment name](/docs/concepts/overview/working-with-objects/names#path-segment-names).
```

Comment:
@tengqm , The change seems fine. What do you think about wrapping the sentence in a note shortcode?